css/styles.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  width: 60%;
+  padding: 1.5rem;
+}
+
+.header-logo {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  padding-top: 2rem;
+  padding-left: 20px;
+}
+
+.logo {
+  margin-top: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.hamburg-logo {
+  padding-right: 1.5rem;
+}
+
+.section-header {
+  background-image: url(../Img/Header-img.png);
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  background-size: cover;
+  background-position: 80% 5%;
+}
```

Comment:
Excelent work adding your background image on the section 💯 💯 💯 
After fixing the different elements that you have to fix, kindly fix the background image. Since this one has to cover all the height of the view.
- [ ] To do this, you can use a `height` 100vh on your section that includes the background image, in this way the height of your image will cover the view screen 💯 💯 💯 

![image](https://user-images.githubusercontent.com/93630700/206809176-68dd98de-4763-4438-9787-3dfc64c12095.png)